packages/task/src/browser/taskNode.ts

```diff
@@ -0,0 +1,37 @@
+/********************************************************************************
```

Comment:
Please rename the file based on our [coding guidelines](https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#lower-case-names). In this case it'd be `task-node.ts`. I'm not sure node is appropriate in this context, it is usually reserved in the application when dealing with `trees` and can cause confusion.

Reply:
done, thanks.
Was renamed to task-node.ts.
I used the word "node" because i am building a graph of tasks dependencies , since i want to check there is no circular dependencies in the graph (DAG)